ui-ngx/src/app/modules/home/components/widget/lib/indicator/signal-strength-widget.component.ts

```diff
@@ -389,4 +391,15 @@ export class SignalStrengthWidgetComponent implements OnInit, OnDestroy, AfterVi
     }
   }
 
+  getLowestSignalRange(): number {
+    return this.settings.activeBarsColor.rangeList.reduce((acc: number, colorRange: ColorRange)=>{
+      if (acc < colorRange.from && acc < colorRange.to) {
+        return acc;
+      }
+      if (colorRange.from < colorRange.to) {
+        return colorRange.from;
+      }
+      return colorRange.to;
+    }, -100);
+  }
```

Comment:
How should the widget behave when a constant or a function determines the color?